lib/msf/core/feature_manager.rb

```diff
@@ -31,6 +33,12 @@ class FeatureManager
         name: SERVICEMANAGER_COMMAND,
         description: 'When enabled you will have access to the _servicemanager command',
         default_value: false
+      }.freeze,
+      {
+        name: DATASTORE_FALLBACKS,
+        description: 'When enabled you will can consistently set username across modules, instead of setting SMBUser/FTPUser/BIND_DN/etc',
```

Comment:
```suggestion
        description: 'When enabled you can consistently set username across modules, instead of setting SMBUser/FTPUser/BIND_DN/etc',
```